### ISSUE TITLE
Optionally raise --alert only after completing test

### ISSUE THREAD

**Issue by SimonCMa** [sqlmapproject/sqlmap#5232]
Hello,

I am using SQLMap with the option --alert to have an automatically detectable False/True indicator, whether an SQL injection vulnerability was found for one of the (many) tested API-endpoints. However, I encountered the following problem: it seems that the alert is raised already in the course of running the test, even if the reason later turns out to be a "false positive", i.e. a false alarm.

Is this true or have I analyzed the behavior incorrectly? If correct, it would be a valuable option for me if raising the alert could be postponed to the end of the current injection test, so that false-positives would already be identified as such and thus would not result in an alert.

Thanks for maintaining this project. :)

**Comment by stamparm** [comment]
With the latest revision switched completely to run `--alert` after all tests end (i.e. in case of false-positive, it will be ignored)
